optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/benchmarkReport.html.ftl

```diff
@@ -182,10 +182,38 @@
                                                 <#assign singleBenchmarkResult = solverBenchmarkResult.findSingleBenchmark(problemBenchmarkResult)>
-                                                <#if !singleBenchmarkResult.success>
+                                                <#if !singleBenchmarkResult.hasAllSuccess()>
                                                     <td><span class="label label-important">Failed</span></td>
                                                 <#else>
-                                                    <td>${singleBenchmarkResult.scoreWithUninitializedPrefix}&nbsp;<@addSingleBenchmarkBadges singleBenchmarkResult=singleBenchmarkResult/></td>
+                                                    <#if solverBenchmarkResult.subSingleCount lte 1>
+                                                        <td>${singleBenchmarkResult.averageScoreWithUninitializedPrefix!""}&nbsp;<@addSolverProblemBenchmarkResultBadges solverProblemBenchmarkResult=singleBenchmarkResult/></td>
+                                                    <#else>
+                                                        <td><div class="dropdown">
+                                                            <span class="nav nav-pills dropdown-toggle" id="dLabel" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                                                ${singleBenchmarkResult.averageScoreWithUninitializedPrefix!""}&nbsp;<@addSolverProblemBenchmarkResultBadges solverProblemBenchmarkResult=singleBenchmarkResult/>
+                                                                <span class="caret"></span>
+                                                            </span>
+                                                            <ul class="dropdown-menu" aria-labelledby="dLabel">
+                                                                <li class="dropdown-header"><strong>Individual runs</strong></li>
+                                                                <li role="separator" class="divider"></li>
+                                                                <#list singleBenchmarkResult.subSingleBenchmarkResultList as subSingleBenchmarkResult>
+                                                                    <li class="dropdown-header"><strong>Run #${subSingleBenchmarkResult.getSubSingleBenchmarkIndex()}</strong></li>
+                                                                    <li>${subSingleBenchmarkResult.scoreWithUninitializedPrefix!""}&nbsp;<@addSolverProblemBenchmarkResultBadges solverProblemBenchmarkResult=subSingleBenchmarkResult/></li>
+                                                                </#list>
+                                                                <li role="separator" class="divider"></li>
+                                                                <li class="dropdown-header"><strong>Average</strong></li>
```

Comment:
We should probably look into replacing these strong usages with a CSS class, but let's worry about that in a different jira, as this probably the tip of the iceberg.